frodo.js

```diff
@@ -2,12 +2,13 @@
  * Extras Plugin
```

Comment:
SCRIPT-Well done with the efficient error handling, it covers all the possible scenarios.